app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -1,13 +1,10 @@
-/* Placeholder for backend dummy app */
 (function($) {
   SpreePayuIntegration = {
-    updateSaveAndContinueVisibility: function(e) {
+    updatePayuSelectedClass: function(e) {
```

Comment:
as I already said, I do not get what gain gives those two methods. And simplification like:

instead of this:
```
updatePayuSelectedClass: function(e) {
      if (this.isPayuChoosen(e)) {
        this.addPayuSelectedClass();
      } else {
        this.removePayuSelectedClass();
      }
    },
```

doing this:

```
updatePayuSelectedClass: function(e) {
     $('#checkout_form_payment').toggleClass('payu_selected', this.isPayuChoosen(e));
    },
```
and removing those extra methods: `addPayuSelectedClass`, `removePayuSelectedClass`.